linkerd/telemeter/usage/src/main/scala/io/buoyant/linkerd/telemeter/UsageDataTelemeter.scala

```diff
@@ -79,13 +81,22 @@ private[telemeter] object UsageDataTelemeter {
   def mkContainerManager: Option[String] =
     sys.env.get("MESOS_TASK_ID").map(_ => "mesos")
 
+  def mkStartTime(metrics: Map[String, Number]): Option[String] = {
+    val tz = TimeZone.getTimeZone("UTC")
+    val df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm'Z'")
+    df.setTimeZone(tz)
+    val uptime = metrics.get("jvm/uptime").map(_.intValue()).getOrElse(0)
```

Comment:
I think this conflicts with https://github.com/linkerd/linkerd/pull/1047.
Sorry!!! 😢